src/shared/components/carousel/index.js

```diff
@@ -16,53 +16,74 @@ const settings = {
   slidesToShow: 1,
   slidesToScroll: 1,
   prevArrow: <Arrow direction="left"/>,
-  nextArrow: <Arrow direction="right"/>
+  nextArrow: <Arrow direction="right"/>,
+  responsive: [{
+    breakpoint: 769,
+    settings: {
+      centerPadding: '20%'
+    }
+  }, {
+    breakpoint: 481,
+    settings: {
+      centerPadding: '12%'
+    }
+  }]
 }
 
-const Carousel = ({ itemsList, modifier, size, onVideoClick, activeIndex, translationsList, isMobile }) => {
-  let items
-  const numberOfSlidesToShow = isMobile ? 1 : size
-  const finalSettings = { ...settings, slidesToShow: numberOfSlidesToShow }
+class Carousel extends Component {
```

Comment:
Why did you change this to a non functional component?

Reply:
I changed it to a non functional component because I was using an approach that requires this component t be a non-functional component. However, I did it other differently and I forgot to restore this component back to a functional component. 
Thank you!